maven/bnd-export-plugin/src/main/java/aQute/bnd/build/StandaloneRun.java

```diff
@@ -0,0 +1,25 @@
+package aQute.bnd.build;
```

Comment:
What I do here is to open the Run class up to allow to set the run bundles.
So I have to be in the same package. This is only a workaround though. A better solution would be to add a setter in Run like I did in my class. I did not want to mess with other code though.